docs/guide/types/label.md

```diff
@@ -216,7 +220,7 @@ All of these options can be [Scriptable](../options#scriptable-options).
 | `borderColor` | [`Color`](../options#color) | `undefined` | Stroke color of the pointer of the callout.
 | `borderDash` | `number[]` | `[]` | Length and spacing of dashes of callout. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setLineDash).
 | `borderDashOffset` | `number` | `0` | Offset for line dashes of callout. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineDashOffset).
-| `borderJoinStyle` | `string` | `'miter'` | Border line joint style of the callout. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineJoin).
+| `borderJoinStyle` | `string` | `'miter'` | Border line join style of the callout. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineJoin).
```

Comment:
Are you sure `join` is more appropriate word for this? I am not sure either way.

Reply:
In chart.js is described as "Line joint style".

Reply:
Shall I update using that sentence?

Reply:
My guess is `joint` is the proper word here. @etimberg?

Reply:
hmmm, I'm not sure. I've always use "join style" for this.